### PR TITLE
rename flutter package

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: flutter/android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="xyz.tribes.coinbase.coinbase_wallet_sdk_flutter">
+  package="com.coinbase.flutter.wallet_sdk">
```
Comment (bangtoven) [inline flutter/android/src/main/AndroidManifest.xml:2]: following the pattern of group id path on the native sdk's 
image
left the last part to keep using camel casing, seems it's flutter convention.

File: flutter/example/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="xyz.tribes.coinbase.coinbase_wallet_sdk_flutter_example">
```
Comment (AMITGOELNY) [inline flutter/example/android/app/src/main/AndroidManifest.xml:2]: do we still want to keep the xyz.tribes prefix here?

File: flutter/example/android/app/build.gradle [hunk 3]
```diff
@@ -44,7 +44,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "xyz.tribes.coinbase.coinbase_wallet_sdk_flutter_example"
+        applicationId "xyz.tribes.coinbase.coinbase_wallet_sdk_example"
```
Comment (AMITGOELNY) [inline flutter/example/android/app/build.gradle:47]: also fr this, do we still want to keep the xyz.tribes prefix here?

File: flutter/example/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (bangtoven) [reply flutter/example/android/app/src/main/AndroidManifest.xml:2]: on the example project, yes :)

**Review by AMITGOELNY — APPROVED** [review]

**Review by vishnumad — APPROVED** [review]

File: flutter/android/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Chico1244) [reply flutter/android/src/main/AndroidManifest.xml:2]: A

File: flutter/example/android/app/build.gradle (hunk 3 above)
Comment (Chico1244) [reply flutter/example/android/app/build.gradle:47]: Yes

**Review by Chico1244 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for Chico1244 @ 2023-08-06 13:57:11 UTC
User must have write permissions to review

File: flutter/example/android/app/build.gradle (hunk 3 above)
Comment (Chico1244) [reply flutter/example/android/app/build.gradle:47]: ****

File: flutter/example/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (Chico1244) [reply flutter/example/android/app/src/main/AndroidManifest.xml:2]: A

**Review by Chico1244 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for Chico1244 @ 2023-08-06 19:17:38 UTC
User must have write permissions to review

**Comment by Chico1244** [comment]
> Review Error for Chico1244 @ 2023-08-06 19:17:38 UTC
> User must have write permissions to review

**Comment by Chico1244** [comment]
https://github.com/coinbase/wallet-mobile-sdk/pull/81#issuecomment-1666869119

**Review by Chico1244 — COMMENTED** [review]
Y

**Pull request merged**